### PR TITLE
Projects Section Cards Script - Modal Details Popup

### PR DESCRIPTION
**Pull request by raminka13**
💯 **This Project was really CHALLENGING.

- We implement the user clicks (or taps) the button project details, and the popup with details appears. ⚙️
- `HTML`close (X) button made of `div` and 3 `span`. Colors in `CSS`. 🧩
- Implement popups for both mobile and desktop screen sizes. 👾
- `JavaScript` array to store all of the information for all projects. ⌘

### CODE REVIEW

File: CSS/Styles.css [hunk 1]
```diff
@@ -317,48 +339,158 @@ ul {
...
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  box-sizing: border-box;
+  background-color: #fff;
+  margin: auto;
+  padding: 0;
+  width: 99%;
+  height: 99%;
+}
```
Comment (ShahierNashaat) [inline CSS/Styles.css]: - [ ] Please follow the Figma design for the project popup window in the desktop size, as the buttons and the description should have a white background and there is a horizontal scroll in your design please fix that as the responsive design shouldn't have a horizontal scroll.

Your Design | Figma Design
---------------- | -------------------
Screenshot from 2022-06-16 22-02-16 | Screenshot from 2022-06-16 22-01-04

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @raminka13 and @karlavdelgadof , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/Styles.css (hunk 1 above)
Comment (raminka13) [reply CSS/Styles.css]: The figma template for the modal requires scroll on Y axis. It is even a taller aspect ratio, so it's not possible to make it responsive and wider as in any computer display without scrolling. I did my best trying to adapt the design in a pleasant way. The buttons out of the container were a mistake I'm fixing now.
Thanks.

Comment (ShahierNashaat) [reply CSS/Styles.css]: By horizontal I mean X axis.

Comment (raminka13) [reply CSS/Styles.css]: __You are totally right. I'm sorry. This project has been very taxing on me.__

**Review by BenMukebo — APPROVED** [review]
### Hi @raminka13 , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL ] Please update your Readme, currently, some headings are missing such as  

``` ##  Getting Started ```

- [ OPTIONAL] Kindly, it would be nice to add explicit steps for cloning and running your app, as this will help you, non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd..`, etc.

- [OPTIONAL] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```
- Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**